model/src/main/scala/explore/model/AppConfig.scala

```diff
@@ -12,7 +12,7 @@ import io.circe._
 import io.circe.generic.semiauto._
 import sttp.model.Uri
 
-case class AppConfig(environment: ExecutionEnvironment, exploreDBURI: Uri, odbURI: Uri)
+case class AppConfig(environment: ExecutionEnvironment, /*exploreDBURI: Uri,*/ odbURI: Uri)
```

Comment:
I wonder if we should just keep it, for once it will make the merge with `auth` harder and OTOH we'll start using the userdb soon

Reply:
Ok, I'll uncomment then. The ExploreDB we end up using will be very different to the one we were using until now, but let's just keep the connection open for now.